app/queries/keywords_query.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class KeywordsQuery
+  attr_reader :current_user, :filter_params
+
+  def initialize(current_user, filter_params)
+    @current_user = current_user
+    @filter_params = filter_params
+  end
+
+  def call
+    return unless filter_params[:adwords_url_contains]
+
+    get_keywords_has_ads_top_urls_contains_word(filter_params[:adwords_url_contains])
```

Comment:
```suggestion
    filter_ads_top_urls
```

You can leverage OOP by letting that method get the appropriate filter, so calling this method is easier (i.e. the responsibility of filtering the `ads_top_urls` is totally handled by the method).